test/Altinn.AccessManagement.Tests/Controllers/V2RightsInternalControllerTest.cs

```diff
@@ -0,0 +1,156 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Altinn.AccessManagement.Controllers;
+using Altinn.AccessManagement.Core.Models;
+using Altinn.AccessManagement.Models;
+using Altinn.AccessManagement.Tests.Fixtures;
+using Altinn.AccessManagement.Tests.Scenarios;
+using Altinn.AccessManagement.Tests.Seeds;
+using Xunit;
+
+namespace Altinn.AccessManagement.Tests.Controllers;
+
+/// <summary>
+/// <see cref="RightsInternalController"/>
+/// </summary>
+public class V2RightsInternalControllerTest(WebApplicationFixture fixture) : ControllerTest(fixture)
+{
+    private static Action<AcceptanceCriteriaTest> WithAssertDbLastDelegationToUserIsRevoked(IParty from, IUserProfile to) => test =>
+    {
+        test.ApiAssertions.Add(async fixture =>
+        {
+            var delegations = await fixture.Postgres.ListDelegationsChanges(filter => filter.Where(delegation => from.Party.PartyId == delegation.OfferedByPartyId && to.UserProfile.UserId == delegation.CoveredByUserId));
+
+            Assert.True(
+                delegations.OrderBy(delegation => delegation.Created).First().DelegationChangeType == DelegationChangeType.RevokeLast,
```

Comment:
OrderByDescending if you want the latest row created.

Reply:
nice catch! done